manifests/config/apache.pp

```diff
@@ -105,7 +105,7 @@
   Array[Stdlib::Fqdn] $serveraliases = [],
   Stdlib::Port $server_port = 80,
   Stdlib::Port $server_ssl_port = 443,
-  Stdlib::HTTPUrl $proxy_backend = 'http://localhost:3000/',
+  Pattern['^(http?|unix)://'] $proxy_backend = 'unix:///run/foreman.sock|http://localhost/',
```

Comment:
```suggestion
  Pattern['^(https?|unix)://'] $proxy_backend = 'unix:///run/foreman.sock|http://localhost/',
```

Reply:
Why `https` here? Doesn't that rule out binding to `0.0.0.0` or `localhost` without https? (i.e. the way we were deploying previously)

Or are you trying to enforce either deploying via unix socket or running foreman service bound to localhost/0.0.0.0 with Puma performing SSL?

Reply:
If you don't want to allow https, then you can leave out `?`
```suggestion
  Pattern['^(http|unix)://'] $proxy_backend = 'unix:///run/foreman.sock|http://localhost/',
```

However, a HTTPS backend is technically valid and IMHO the data type shouldn't forbid it. So while it's unlikely, I'd just allow it in the data type.